tron/mcp.py

```diff
@@ -53,30 +56,34 @@ def reconfigure(self, namespace=None):
             log.exception(f"reconfigure failure: {e.__class__.__name__}: {e}")
             raise e
 
-    def _load_config(self, reconfigure=False, namespace_to_reconfigure=None):
+    def _load_config(self, reconfigure=False, namespace_to_reconfigure=None, tron_start_time=None):
         """Read config data and apply it."""
         with self.state_watcher.disabled():
             self.apply_config(
                 self.config.load(),
                 reconfigure=reconfigure,
                 namespace_to_reconfigure=namespace_to_reconfigure,
+                tron_start_time=tron_start_time,
             )
 
-    def initial_setup(self):
+    def initial_setup(self, tron_start_time=None):
         """When the MCP is initialized the config is applied before the state.
         In this case jobs shouldn't be scheduled until the state is applied.
         """
-        self._load_config()
+        # The job schedules have been applied in first step
```

Comment:
what does first step refer to here?

Reply:
to load_config function, we first load the configs and apply them then second step would be the restore_state function

Reply:
will change it to ``The job schedule factories will be created in the function below``